db/db_bloom_filter_test.cc

```diff
@@ -171,20 +172,30 @@ TEST_F(DBBloomFilterTest, GetFilterByPrefixBloomCustomPrefixExtractor) {
 
     ASSERT_EQ("foo", Get("barbarbar"));
     ASSERT_EQ(TestGetTickerCount(options, BLOOM_FILTER_USEFUL), 0);
+    // TODO (Zhongyi): uncomment the asserts involving level_to_perf_context when per
+    // level perf context is enabled in block based table reader
+    // ASSERT_EQ(0, (*(get_perf_context()->level_to_perf_context))[0].bloom_filter_useful);
```

Comment:
With all those commented out, the unit tests won't be able to catch any regression bug of per level perf context. I don't think it is good.

Reply:
You can manually call the MACRO to set the counters and verify them.
Also, it's not a good idea to have counters that are never set. It confuses people. I suggest the follow-up PR that correctly set those counters be committed as soon as possible.

Reply:
added test coverage in `PerfContextTest.PerfContextByLevelGetSet` to do set and get of per level perf context